packages/main/src/Menu.hbs

```diff
@@ -80,11 +81,13 @@
 						part="icon"
 						name="slim-arrow-right"
 						class="ui5-menu-item-icon-end"
+						?rtl="{{../isRtl}}"
```

Comment:
Are these ?rtl needed as the styles are now refactored to work without the "rtl" attrbiute

Reply:
You're right, removed.